content/_data/events/scale15x.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "Scale 15x"
+date: "2017-03-02 thru 03-05T09:00:00"
```

Comment:
I doubt that works. It needs to be a time stamp so the site can parse it into those calendar sheets.